grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -935,13 +937,16 @@ class OrderService {
         }
 
         importedLines.each { line ->
-            if (line.quantityToShip > 0) {
+            if ((line.quantityToShip as int) > 0) {
                 OrderItem orderItem = order.orderItems.find { it.id == line.id }
                 Product product = line.productCode ? Product.findByProductCode(line.productCode) : orderItem.product
 
                 InventoryItem inventoryItem = null
-                if (line.lotNumber && line.expiration) {
-                    inventoryItem = inventoryService.findOrCreateInventoryItem(product, line.lotNumber, line.expiration)
+                def expiry = null
```

Comment:
Why expiry is set to null here and then used in inventoryService.findOrCreateInventoryItem(product, line.lotNumber, expiry)?

Reply:
Before doing findOrCreate the value of expiry is assigned. And I declared it here because I reference this later (if the inventory item is created, ofc). This is here just to get rid of an IDE warning.